index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Frederick's-Portfolio</title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link rel="stylesheet" href="styles.css">
+
+</head>
+<body>
+       <nav class="nav-bar">
+        <h1 class="Logo">My Logo</h1>
+        <img src="Images/Union.png" alt="Three-Lines">
+       </nav>
+
+       <section class="headline">
+        <div class="Greetings-txt">
+           <h2 class="cheers-txt">I'm Frederick Glad to see you!</h2>
+        </div>
+
+        <div class="main-headline-txt">
+          <p>
+            I am a beginner student in the Microverse Program, this is my first week!
+            I'm learning right now how to implement flexbox properties in CSS. 
+            This is my first project using flexbox and setting up a mobile version skeleton
+          </p>
+
+          <div class="connect">
+               <h3 class="small-txt">LET'S CONNECT</h3>
+          </div>
+
+          <div class="media-Images">
+               <img src="Images/Twitter-icon.svg"   alt="Twitter Icon">
+               <img src="Images/Linkedin-icon.svg"   alt="Linkedin Icon">
+               <img src="Images/Elegant-M-Icon.svg"   alt="Micro Icon">
+               <img src="Images/GitHub-Desktop-icon.svg" alt="Github Icon">
+               <img src="Images/Peace-Emoji-Icon.svg"  alt="Peace Icon">
+          </div>
+
```

Comment:
- [ ] Please must use a `ul` tag to group the list of icons, and `li` per icon; each icon has a link (`a` tag) 
as it is semantically better this in this way